cocos/core/platform/sys.ts

```diff
@@ -259,6 +259,11 @@ export const sys = {
         return true;
     },
 
+    /**
+     * @engineInternal
+     */
+    __isWebIOS14OrIPadOS14Env: false,
```

Comment:
Why need it?

Reply:
we dynamically inject property `__isWebIOS14OrIPadOS14Env` into sys object,
it's better to declare the property forward